src/Handlers/ShortcodeParametersHandler.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module\Handlers;
```

Comment:
@mecha, I believe this is your standard. Don't you think that these kind of namespaces look a bit weird? Why not the classic "vendor/module" approach?

Reply:
More of a convention, rather than a standard. I adopted this to segregate module-specific classes from the ones installed by dependency packages that share the same "vendor/package" namespace. And yes they do look a bit weird.

What exactly would be "module" part be in "vendor/module"? Can you give an example using this module's name?

Reply:
Yea, something like `RebelCode\WpBookingsShortcode` could be a good namespace in this case. What do you think?

Reply:
Works for me.

Reply:
@roman, could you please change the root namespace of this module accordingly? Same for all other modules, where applicable and feasible right now.